boards/deprecated.cmake

```diff
@@ -916,3 +916,6 @@ set(yd_esp32_DEPRECATED
 set(mimx8mp_phyboard_pollux/mimx8ml8/m7_DEPRECATED
     phyboard_pollux/mimx8ml8/m7
 )
+set(mimx8mm_phyboard_polis/mimx8mm6/m4_DEPRECATED
```

Comment:
line 388 needs updating too

Reply:
Oh, right! The old deprecation that has been introduced when HWMv2 came must directly point to the most recent board. Will change.

Changed also l.368 (Polis).